client/elements/page-selector.html

```diff
@@ -234,14 +243,21 @@
               this.addEventListener('par-menu-copied', (e) => {
                   let success = '';
                   if (e.detail.success) {
```

Comment:
I think the variable name should be something more like result. Success suggests binary value rather than string value

Reply:
I think this is your code